supply-chain/config.toml

```diff
@@ -34,6 +34,18 @@ audit-as-crates-io = false
 [policy.quickjs-wasm-sys]
 audit-as-crates-io = false
 
```

Comment:
Temporary until we're off the fork; I intended to add a comment but `cargo vet` didn't liked it.